src/robots.ts

```diff
@@ -22,9 +22,15 @@ export function robotsTxt(request: Request, env: CFEnvironment) {
   const url = Url.fromRequest(request)
   if (url.pathname !== '/robots.txt') return null
 
+  const sitemap =
+    isInstance(url.subdomain) && url.subdomain === Instance.De
```

Comment:
nit: also nice for the boolean to be declared as a well named variable, then the ternary logic is super clear